java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -775,10 +777,10 @@ class LocalVariableDeclStmt extends Stmt, @localvariabledeclstmt {
   /** Gets an index of a variable declared in this local variable declaration statement. */
   int getAVariableIndex() { exists(getVariable(result)) }
 
-  /** Gets a printable representation of this statement. May include more detail than `toString()`. */
   override string pp() { result = "local variable declaration" }
 
-  /** This statement's Halstead ID (used to compute Halstead metrics). */
+  override string toString() { result = "local variable declaration" }
```

Comment:
The syntax here is either `var ...` or `<type> ...`, but maybe we should just use the `var ...` as the toString in all cases. Getting a suitable `toString` of the type involves additional recursion, but might be ok - it just seems a bit much if a fixed string does the job.

Reply:
Yeah, that's probably a good idea.